cogsmisc/stats.py

```diff
@@ -177,6 +177,19 @@ async def get_guild_count(bot):
         cluster_servers = await bot.rdb.get_whole_dict(GUILD_RDB_KEY)
         return sum(int(v) for v in cluster_servers.values())
 
+    @staticmethod
```

Comment:
Analytics: tracks, for each Discord user,
- their DDB user ID
- how many times the link has been used
- the last time the link was used
- the first time the link was used